test/test_red_black_tree.jl

```diff
@@ -0,0 +1,71 @@
+include("../src/red_black_tree.jl")
+@testset "RBTree" begin
+    t = RBTree{Int}()
+    for i = 1:10000
+    	insert!(t, i)
+    end
+    for i = 1:10000
+    	@test search_key(t, i)
+    end
+
+    for i = 10001:20000
+        @test !search_key(t, i)
+    end
+
+    for i = 1:2:10000
+    	delete!(t, i)
+    end
+
+    for i = 1:10000
+    	if iseven(i)
+    		@test search_key(t, i)
+    	else
+    		@test !search_key(t, i)
+    		@test_throws KeyError delete!(t, i)
+    	end
```

Comment:
the indentation in this bit is all messed up.
it is a mix of 4 spaces, 2 spaces, and tabs

Reply:
Oops.